src/main/scala/firrtl/transforms/GroupComponents.scala

```diff
@@ -119,6 +119,11 @@ class GroupComponents extends firrtl.Transform {
       }
     }
 
+    // Unused nodes are reachable from any group nor the root, add them to root group
```

Comment:
Minor quibble, but I don't understand this comment

Reply:
The comment is missing an `not` in front of `reachable`, should be:
`Unused nodes are not reachable from any group nor the root, add them to root group`

`reachableNodes` is a Map from labels to the nodes that label can reach. The "default" or "root" label (in the code as `""`) should retain any nodes that are not part of one of the groups.

Since the code is using DiGraph reachability, nodes that are not connected to anything don't get put in any of the groups; thus giving an error when they're looked up later.

Have any ideas on how to better communicate this?

Reply:
I think adding the missing *not* is enough, I thought that was what it was supposed to be but wasn't sure

Reply:
I updated the comment